resources/baseResource.ts

```diff
@@ -12,7 +12,7 @@ export class BaseResource {
         this.headers = {
             "Authorization": `Bearer ${token}`,
             "Content-Type": "application/vnd.api+json",
-            "User-Agent": "unit-node-sdk"
+            ...(config?.shouldSkipUserAgent ? {} : { "User-Agent": "unit-node-sdk" }),
```

Comment:
please change to:
...(config?.sdkUserAgent && { "User-Agent": "unit-node-sdk" })